tests/src/Kernel/EntityReference/Views/OgStandardReferenceRelationshipTest.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\EntityReference\Views\OgStandardReferenceRelationshipTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\EntityReference\Views;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\entity_test\Entity\EntityTestMul;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\views\Kernel\ViewsKernelTestBase;
+use Drupal\views\Tests\ViewTestData;
+use Drupal\views\Views;
+
+/**
+ * Tests OG standard reference relationship data.
+ *
+ * @group og
+ *
+ * @see core_field_views_data()
+ */
+class OgStandardReferenceRelationshipTest extends ViewsKernelTestBase {
+
+  /**
+   * Views used by this test.
+   *
+   * @var array
+   */
+  public static $testViews = [
+    'test_og_standard_reference_entity_test_view',
+    'test_og_standard_reference_reverse_entity_test_view',
+    'test_og_standard_reference_entity_test_mul_view',
+    'test_og_standard_reference_reverse_entity_test_mul_view',
+  ];
+
+  /**
+   * Modules to install.
+   *
+   * @var array
+   */
+  public static $modules = ['user', 'field', 'entity_test', 'views', 'og_standard_reference_test_views', 'og'];
+
+  /**
+   * The entity_test entities used by the test.
+   *
+   * @var array
+   */
+  protected $entities = [];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp($import_test_views = TRUE) {
+    parent::setUp();
+
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('entity_test');
+    $this->installEntitySchema('entity_test_mul');
+
+    // Create reference from entity_test to entity_test_mul.
+    $this->createEntityReferenceField('entity_test', 'entity_test', 'field_test_data', 'field_test_data', 'entity_test_mul');
```

Comment:
It's copied from core but that helper function can totally use the Og helper to create the field!